src/sentry/mediators/sentry_app_installations/destroyer.py

```diff
@@ -34,11 +36,19 @@ def _destroy_service_hooks(self):
 
     def _destroy_installation(self):
         if self.notify:
-            InstallationNotifier.run(
-                install=self.install,
-                user=self.user,
-                action='deleted',
-            )
+            try:
+                InstallationNotifier.run(
+                    install=self.install,
+                    user=self.user,
+                    action='deleted',
+                )
+            except Exception as exc:
+                capture_exception(exc)
+                # if the timeout is not due to timeout or connectivity, then raise it to
```

Comment:
bb nitpick I think you meant `if the exception is not due to...`

Reply:
@MeredithAnya fixed